addons/vehicles_dragoon/CfgWeapons.hpp

```diff
@@ -0,0 +1,159 @@
+class CfgWeapons {
+    class CannonCore;
+    class autocannon_Base_F: CannonCore {
+        class player;
+    };
+    class autocannon_30mm_CTWS: autocannon_Base_F
+    {
+        displayName = "$STR_A3_GATLING_30MM_CTWS0";
+        muzzles[] = {"HE","AP"};
+        class HE: autocannon_Base_F
+        {
+            displayName = "$STR_A3_GATLING_30MM_CTWS0";
+            magazines[] = {"140Rnd_30mm_MP_shells","140Rnd_30mm_MP_shells_Tracer_Red","140Rnd_30mm_MP_shells_Tracer_Green","140Rnd_30mm_MP_shells_Tracer_Yellow","78Rnd_30mm_MP_shells_Tracer_Red","78Rnd_30mm_HEI_shells_Tracer_Red"};
```

Comment:
Do we change or append to this array? I'd we append only include the added magazine else this will cause other boxes to stop working:

X[] += {"thing"};

To append to array.